binaryRepresentation/binaryRepresentation/tests.c

```diff
@@ -0,0 +1,91 @@
+#include "binaryRepresentation.h"
+
+#include <stdio.h>
+#include <stdbool.h>
+
+#define INT_SIZE 8
+#define TWO_POWER_EIGHT 256
+
+_Bool convertToBinTest() {
```

Comment:
```suggestion
bool convertToBinTest() {
```
Если Вы в своём коде используете что-то, начинающееся с подчёркивания, это в подавляющем большинстве случаев означает, что Вы что-то делаете не так.